duration max_duration
---
float64 snr